pkg/devicetemplateconverter/devicetemplateconverter.go

```diff
@@ -88,5 +105,7 @@ func (dtc *DeviceTemplateConverter) RegisterServices(s *grpc.Server) {
 
 // RegisterHandlers registers gRPC handlers.
 func (dtc *DeviceTemplateConverter) RegisterHandlers(s *runtime.ServeMux, conn *grpc.ClientConn) {
-	ttnpb.RegisterEndDeviceTemplateConverterHandler(dtc.Context(), s, conn)
+	if err := ttnpb.RegisterEndDeviceTemplateConverterHandler(dtc.Context(), s, conn); err != nil {
+		panic(err)
+	}
```

Comment:
Why?

Reply:
I was running the golintci locally to catch errors that would show up on the CI, one of them was not checking the errors of methods called in the `RegisterHandlers`. The panic was due to this being called at the server initialization, so it made sense to me stopping the server if the handlers aren't weren't able to be registered.

Nevertheless the other places indeed don't do this so might not be appropriate to do so here, will remove it.